docs/data/joy/getting-started/templates/ios/theme.tsx

```diff
@@ -0,0 +1,632 @@
+          tertiary: '#2C2C2E',
+        },
+        groupedBackground: {
+          primary: '#000000',
+          secondary: '#1C1C1E',
+          tertiary: '#2C2C2E',
+        },
+        label: {
+          primary: '#ffffff',
+          secondary: 'rgba(235 235 245 / .60)',
+          tertiary: 'rgba(235 235 245 / .30)',
+          quarternary: 'rgba(235 235 245 / .18)',
+        },
+        fill: {
+          primary: 'rgba(120 120 128 / .36)',
+          secondary: 'rgba(120 120 128 / .32)',
+          tertiary: 'rgba(120 120 128 / .24)',
+          quarternary: 'rgba(120 120 128 / .18)',
+        },
+        separator: {
+          opaque: '#38383A',
+          nonOpaque: 'rgba(84 84 88 / .65)',
+        },
+      },
+    },
+  },
+  fontFamily: {
+    body: '"SF Pro", var(--ios-fontFamily-fallback)',
+  },
+  fontSize: {
+    xl3: '34px',
+    xl2: '28px',
+    xl: '22px',
+    lg: '20px',
+    md: '17px',
+    sm: '16px',
+    xs: '15px',
+    xs2: '14px',
+    xs3: '12px',
+  },
+  materials: {
+    thick: {
+      background: `linear-gradient(rgba(255 255 255 / 0.78), rgba(255 255 255 / 0.78))
+        ,linear-gradient(rgba(255 255 255 / 0.34), rgba(255 255 255 / 0.34))`,
+      backdropFilter: 'blur(50px)',
+      [`[data-joy-color-scheme="dark"] &`]: {
+        background: 'rgba(0 0 0 / 0.6)',
+      },
+    },
+    regular: {
+      background: `linear-gradient(rgba(255 255 255 / 0.39), rgba(255 255 255 / 0.39))
+      ,linear-gradient(rgba(255 255 255 / 0.27), rgba(255 255 255 / 0.27))`,
+      backdropFilter: 'blur(50px) saturate(121%)',
+      [`[data-joy-color-scheme="dark"] &`]: {
+        background: 'rgba(0 0 0 / 0.41)',
+        backdropFilter: 'blur(50px)',
+      },
+    },
+    thin: {
+      background: `linear-gradient(rgba(255 255 255 / 0.51), rgba(255 255 255 / 0.51))
+      ,linear-gradient(rgba(255 255 255 / 0.05), rgba(255 255 255 / 0.05))`,
+      backdropFilter: 'blur(50px) saturate(145%)',
+      [`[data-joy-color-scheme="dark"] &`]: {
+        background: 'rgba(0 0 0 / 0.26)',
+        backdropFilter: 'blur(50px)',
+      },
+    },
+    ultrathin: {
+      background: `linear-gradient(rgba(255 255 255 / 0.12), rgba(255 255 255 / 0.12))
+      ,linear-gradient(rgba(255 255 255 / 0.11), rgba(255 255 255 / 0.11))`,
+      backdropFilter: 'blur(30px)',
+      [`[data-joy-color-scheme="dark"] &`]: {
+        background: 'rgba(0 0 0 / 0.1)',
+        backdropFilter: 'blur(50px)',
+      },
+    },
+    navbar: {
+      backgroundColor: 'rgba(249 249 249 / 0.94)',
+      boxShadow: '0 0.33px 0 0 rgba(0 0 0 / 0.3)',
+      backdropFilter: 'blur(20px)',
+      [`[data-joy-color-scheme="dark"] &`]: {
+        backgroundColor: 'rgba(29 29 29 / 0.94)',
+        boxShadow: '0 0.33px 0 0 rgba(255 255 255 / 0.15)',
+        backdropFilter: 'blur(20px) saturate(130%)',
+      },
+    },
+  },
+  unstable_sxConfig: {
+    material: {
+      style: (props) => props.theme.materials?.[props.material],
+    },
+  },
```

Comment:
@mnajdova Got a nice use case for this!

Reply:
Cool, would be great if you can share more feedback (related to things that can be improved) :)